extension/scripts/global/globalClasses.js

```diff
@@ -38,10 +37,11 @@ class FeatureManager {
 
 		let collapsed = this.containerID in filters.containers ? filters.containers[this.containerID] : false;
 
-		document.find(".content").appendChild(
+		document.find("body").appendChild(
```

Comment:
`document.body` would be better, in this case a find is not needed

Reply:
I always forget that :D